### DISCUSSION TITLE
[distribution] helix not launching (installed via msys2)

### DISCUSSION THREAD

**Discussion by goyalyashpal** [helix-editor/helix#8017] [Troubleshooting] [answered]
### Summary

when installed via msys2, helix is not launching.

### Reproduction Steps

 

[helix docs]: https://docs.helix-editor.com/install.html#msys2

I tried this as per instructions at [helix docs]

```console
$ pacboy debug -S helix:u
:: Executing pacman --color auto -S mingw-w64-ucrt-x86_64-helix...
  ... MORE ...

$ # helix's executable name elsewhere is hx; following from that expectation
$ hx
bash: hx: command not found

$ helix
/ucrt64/bin/helix: line 3: /lib/helix/hx.exe: No such file or directory
```

#### I expected this to happen:

`hx` to launch helix.

#### Instead, this happened:

the `hx` command is not found, and with `helix`, it says: `/ucrt64/bin/helix: line 3: /lib/helix/hx.exe: No such file or directory`

### Helix log

N/A

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Windows 10 22H2

### Terminal Emulator

N/A

### Helix Version

23.05-2

**Comment by pascalkuthe** [comment]
This is an issue with the nysys2 package you need to raise it on their issue tracker. We do not maintain package manager pakcgaes.

**Reply by goyalyashpal** [reply, marked as answer]
> This is an issue with the nysys2 package

true. it requires setting the `$MINGW_PREFIX` prefix

but strangely enough, `neovim` from same arch/prefix  `mingw-w64-x86_64-` launches directly as `nvim` without requiring any of this

https://github.com/msys2/MINGW-packages/issues/17797#issuecomment-1686531648
